cabal.project.release

```diff
@@ -1,4 +1,8 @@
+packages: Cabal-QuickCheck/
+packages: Cabal-described/
 packages: Cabal-syntax/
+packages: Cabal-tests/
+packages: Cabal-tree-diff/
```

Comment:
I think the dev packages were omitted on purpose, and what this job should be doing is taking only the main 4 packages and dry-running a build with tests and benchmarks disabled. That's, more or less, what is happening in https://github.com/haskell/cabal/blob/master/.gitlab/ci.sh and that's what needs testing IMO.

Reply:
Oops sorry about that. I see my mistake, having `--enable-tests --enable-benchmarks` on the command line that overrides the project setting. I like the fish shell but will have to be more careful with [suggestions](https://fishshell.com/docs/current/interactive.html#autosuggestions) and reusing the command history.

Reply:
And worse still, I'd copied that command line with `--enable-tests --enable-benchmarks` into the workflow!

Reply:
Happens to me all the time! (I use fish too.)